### ISSUE TITLE
module-info causes duplicate-dependency-class to fail

### ISSUE THREAD

**Issue by ajoberstar** [nebula-plugins/gradle-lint-plugin#139]
It appears the presence of a `module-info.class` in the JAR is triggering the `duplicate-dependency-class` rule to fail. I'm guessing these will have to be whitelisted somehow in the rule.

Reproduce with:

**build.gradle**
```groovy
plugins {
  id 'nebula.lint' version '7.9.5'
  id 'java'
}

gradleLint.rules = ['duplicate-dependency-class']

repositories {
  jcenter()
}

dependencies {
  compile 'org.slf4j:slf4j-api:1.8.0-alpha2'
  testRuntime 'org.slf4j:slf4j-simple:1.8.0-alpha2'
}

wrapper {
  gradleVersion = '4.0.2'
}
```

**src/main/java/example/MyClass.java**
```groovy
package example;

import org.slf4j.LoggerFactory;

public class MyClass {
        public void stuff(String msg) {
                LoggerFactory.getLogger(MyClass.class).info(msg);
        }
}
```

**Running:**

```
gradle wrapper
gradlew classes --info
```

**Outputs:**

```
:autoLintGradle (Thread[Task worker,5,main]) started.

> Task :autoLintGradle
Putting task artifact state for task ':autoLintGradle' into context took 0.0 secs.
Executing task ':autoLintGradle' (up-to-date check took 0.0 secs) due to:
  Task has not declared any outputs.
org.slf4j:slf4j-api:1.8.0-alpha2 in configuration ':compile' has 1 classes duplicated by org.slf4j:slf4j-simple:1.8.0-alpha2. Duplicate classes: [module-info]
org.slf4j:slf4j-api:1.8.0-alpha2 in configuration ':testRuntime' has 1 classes duplicated by org.slf4j:slf4j-simple:1.8.0-alpha2. Duplicate classes: [module-info]

This project contains lint violations. A complete listing of the violations follows.
Because none were serious, the build's overall status was unaffected.

warning   duplicate-dependency-class         org.slf4j:slf4j-api:1.8.0-alpha2 in configuration ':compile' has 1 classes duplicated by org.slf4j:slf4j-simple:1.8.0-alpha2 (use --info for detailed class list) (no auto-fix available)

warning   duplicate-dependency-class         org.slf4j:slf4j-api:1.8.0-alpha2 in configuration ':testRuntime' has 1 classes duplicated by org.slf4j:slf4j-simple:1.8.0-alpha2 (use --info for detailed class list) (no auto-fix available)

? 2 problems (0 errors, 2 warnings)

To apply fixes automatically, run fixGradleLint, review, and commit the changes.


:autoLintGradle (Thread[Task worker,5,main]) completed. Took 0.044 secs.
```

**Comment by DanielThomas** [comment]
This would apply to `package-info` too.
